app/views/pets/show.html.erb

```diff
@@ -1,8 +1,3 @@
-<h1><%= @pet.name %></h1>
-<h3><%= @pet.age %> years old</h3>
-<h3><%= @pet.breed %></h3>
-<h3><%= @pet.adoptable %></h3>
-<h3><%= @pet.shelter_name %></h3>
-
+<%= render partial: "pets/pet_list", locals: { pet: @pet } %>
```

Comment:
If this partial lives in the `pets` directory, you can just call the name of the partial without the `pets` prefix.